### PR TITLE
Add new contributors with profile avatars

### PR DESCRIPTION
**Pull request by rettetdemdativ**
This PR is based on the first one and adds upon those changes.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -215,26 +215,68 @@ <h3 class="center">Contributors</h3>
...
+            <a href="https://github.com/abisz">Simon Reinsperger</a>
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github-images.s3.amazonaws.com/enterprise/2.8/assets/images/help/profile/identicon.png" />
+            Markus Hösel
+          </li>
+          <li>
+            <img class="profile-picture" src="https://github.com/achim-tecklenburg.png" />
+            <a href="https://github.com/achim-tecklenburg">Achim Tecklenburg</a>
+          </li>
```
Comment (robaxelsen) [inline index.html:245]: Could you also add Monika, @calmandniceperson? She is now [in the list](https://github.com/FCCVienna/FCCVienna.github.io/issues/36#issuecomment-334709076). I forgot to add her to the list, and totally my mistake. If you'd rather not, I can add her once this is in master. Will approve for now.

**Review by robaxelsen — APPROVED** [review]

**Comment by robaxelsen** [comment]
LGTM :+1: 

What do you think @DDCreationStudios?

File: index.html (hunk 1 above)
Comment (rettetdemdativ) [reply index.html:245]: Done. :)

**Review by Createdd — APPROVED** [review]
Looks great!

**Pull request merged**